server/src/main/java/org/kaazing/gateway/server/context/resolve/GatewayContextResolver.java

```diff
@@ -735,7 +735,15 @@ private void parseProperties(Node parent, ServiceProperties properties) {
                     }
                 }
                 if (isSimpleProperty) {
-                    properties.put(node.getLocalName(), nodeValue);
+                    // TODO; consider going to dynamically typed objects
+                    // (i.e. allowing Object as properties value)
+                    // For now if we see a property that is a list, we convert to comma separated list
+                    String existingValue = properties.get(node.getLocalName());
+                    if (existingValue == null) {
+                        properties.put(node.getLocalName(), nodeValue);
+                    } else {
+                        properties.put(node.getLocalName(), existingValue + LIST_SEPARATOR + nodeValue);
```

Comment:
We will need to revisit this eventually.  I don't have a better solution off the top of my head